fetch-todolist/src/component/Home.js

```diff
@@ -1,76 +1,115 @@
-import { useState, useEffect } from "react";
-import AddToList from "./AddToDoList";
+import { useState, useEffect, useCallback } from "react";
 import ToDoList from "./ToDoList";
-import "./styling.css";
-import useFetch from "../customHook/useFetch";
 import DeleteFromList from "./DeleteFromList";
+import { useMutation, useQuery, useQueryClient } from "react-query";
+import { getTodos, deleteTodo } from "../services/todos.services.js";
+import "./styling.css";
 
 const Home = () => {
-  const [list, setList] = useState([]);
   const [idToDel, setIdToDel] = useState();
-  const [isBlockHidden, setIsBlockHidden] = useState(false);
   const [isDelBlockHidden, setIsDelBlockHidden] = useState(false);
-  const { loading, error, data } = useFetch();
-  useEffect(() => {
-    setList(data);
-  }, [data]);
+  const pageSize = 4;
+  const [pageNumber, setPageNumber] = useState(1);
+  const [displayTodos, setTodos] = useState([]);
```

Comment:
for array we can prefer to use useReducer,
for Primitive data types we can use useState

Reply:
it was the need of the assignment, we learnt reducer in the subsequent module and we've implemented it.